spec/datetimeformat.html

```diff
@@ -373,15 +373,15 @@ <h1>ToLocalTime ( _t_, _calendar_, _timeZone_ )</h1>
       <emu-alg>
         1. Assert: Type(_t_) is Number.
         1. Apply calendrical calculations on _t_ for the given _calendar_ and _timeZone_ to produce weekday, era, year, month, day, hour, minute, second, and inDST values. The calculations should use best available information about the specified _calendar_ and _timeZone_, including current and historical information about time zone offsets from UTC and daylight saving time rules.
-        1. If the _calendar_ is `"gregory"`, then the calculations of weekday, year, month, day, hour, minute and second must match the algorithms specified in ES2020, <emu-xref href="#sec-overview-of-date-objects-and-definitions-of-abstract-operations"></emu-xref>:
+        1. If the _calendar_ is `"gregory"`, then the calculations of weekday, year, month, day, hour, minute and second must match the algorithms specified in ES2020, <emu-xref href="#sec-overview-of-date-objects-and-definitions-of-abstract-operations"></emu-xref>. In these calculations, use the value _tz_, which is the time value _t_ + _timeZoneOffset_. The value _timeZoneOffset_ must be calculated according to <emu-xref href="#sec-local-time-zone-adjustment">LocalTZA(_t_, *true*)</emu-xre> where `local time zone` is replaced with timezone `timeZone`.
```

Comment:
@FrankYFTang can you review this prose? Unfortunately all timezone offset calculations only refer to "local time" in ECMA-262, which makes it a little awkward to use.

Reply:
pinging @FrankYFTang. Also @sffc for potentially more feedback